### PR TITLE
Update configure-server-endpoints.md

### PR DESCRIPTION
**Pull request by baardhermansen**
Removed an extra line shift that broke a Note notation. 
Removed language specifics in URLs.
Added an escape character so `\\` is displayed as intended. 
Other very minor cleanups of markdown.

### CODE REVIEW

File: microsoft-365/security/defender-endpoint/configure-server-endpoints.md [hunk 1]
```diff
@@ -287,14 +287,13 @@ The following steps are only applicable if you're using a third-party anti-malwa
...
+- We've identified an issue with Windows Server 2012 R2 connectivity to cloud when static TelemetryProxyServer is used **and** the certificate revocation list (CRL) URLs aren't reachable from the SYSTEM account context. Ensure the EDR sensor is updated to version 10.8210.* or later (using [KB5005292](https://support.microsoft.com/topic/microsoft-defender-for-endpoint-update-for-edr-sensor-f8f69773-f17f-420f-91f4-a8e5167284ac)) to resolve the issue. Alternatively, use an different proxy option ("system-wide") that provides such connectivity, or configure the same proxy via the WinInet setting on the SYSTEM account context.
```
Comment (JohanFreelancer9) [inline microsoft-365/security/defender-endpoint/configure-server-endpoints.md]:
```suggestion
- We've identified an issue with Windows Server 2012 R2 connectivity to cloud when static TelemetryProxyServer is used **and** the certificate revocation list (CRL) URLs aren't reachable from the SYSTEM account context. Ensure the EDR sensor is updated to version 10.8210.* or later (using [KB5005292](https://support.microsoft.com/topic/microsoft-defender-for-endpoint-update-for-edr-sensor-f8f69773-f17f-420f-91f4-a8e5167284ac)) to resolve the issue. Alternatively, use a different proxy option ("system-wide") that provides such connectivity, or configure the same proxy via the WinInet setting on the SYSTEM account context.
```

File: microsoft-365/security/defender-endpoint/configure-server-endpoints.md [hunk 2]
```diff
@@ -287,14 +287,13 @@ The following steps are only applicable if you're using a third-party anti-malwa
...
 - On Windows Server 2012 R2, there's no user interface for Microsoft Defender Antivirus. In addition, the user interface on Windows Server 2016 only allows for basic operations. To perform operations on a device locally, refer to [Manage Microsoft Defender for Endpoint with PowerShell, WMI, and MPCmdRun.exe](/microsoft-365/security/defender-endpoint/manage-mde-post-migration-other-tools). As a result, features that specifically rely on user interaction, such as where the user is prompted to make a decision or perform a specific task, may not work as expected. It's recommended to disable or not enable the user interface nor require user interaction on any managed server as it may impact protection capability.
 - Not all Attack Surface Reduction rules are applicable to all operating systems. See [Attack Surface Reduction (ASR) rules](/microsoft-365/security/defender-endpoint/attack-surface-reduction-rules).
 - Operating system upgrades aren't supported. Offboard then uninstall before upgrading. The installer package can only be used to upgrade installations that have not yet been updated with new antimalware platform or EDR sensor update packages.
```
Comment (JohanFreelancer9) [inline microsoft-365/security/defender-endpoint/configure-server-endpoints.md:293]:
```suggestion
- Operating system upgrades aren't supported. Offboard or uninstall them before upgrading. The installer package can only be used to upgrade installations that have not yet been updated with new antimalware platform or EDR sensor update packages.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @baardhermansen, kindly commit the two additional suggested changes. Many thanks.

File: microsoft-365/security/defender-endpoint/configure-server-endpoints.md (hunk 2 above)
Comment (baardhermansen) [inline microsoft-365/security/defender-endpoint/configure-server-endpoints.md:293]: @JohanFreelancer9 One must both offboard (from the service) and uninstall (the client), so the suggested change will not suffice. 
At least as far as i'm aware? Might have to have that verified with someone?

**Review by JohanFreelancer9 — APPROVED** [review]
@baardhermansen, thank you for the clarification. I have deleted my suggestion.

**Pull request merged**